readme.md

```diff
@@ -401,6 +411,23 @@ Default: `false`
 
 If `true`, no quoting or escaping of arguments is done on Windows. Ignored on other platforms. This is set to `true` automatically when the `shell` option is `true`.
 
+### child_process.kill([signal], [options])
```

Comment:
```suggestion
### kill([signal], [options])
```

This should be moved just before `cancel()` in the `README`. That section is about the methods of the returned `child_process`, i.e. only the method name needs to be mentioned.

Reply:
Done: e2d589df448152eefd4eac35c845dc2d4460af69